src/StructurizrPHP/Core/Model/Configuration/Role.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Structurizr for PHP.
+ *
+ * (c) Norbert Orzechowicz <norbert@orzechowicz.pl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace StructurizrPHP\StructurizrPHP\Core\Model\Configuration;
```

Comment:
This should go into `StructurizrPHP\StructurizrPHP\Core\Configuration` namespace, since it's not `Model` but `Workspace` configuration